feat: Add policy id parameter to inspect command (#2174)
cerbos/cerbos: docs/modules/cli/pages/cerbosctl.adoc

You can include disabled policies in the results by adding `--include-disabled` flag.

> `policy-id` flag allows to filter the policies by the policy id while listing.

Reviewer: ```suggestion
You can optionally restrict the list to a specific set of policies by supplying their IDs as trailing arguments. 
```

cerbosctl get dr --name-regexp "^(my|a)_policy\$"
----

> .List derived_roles but filter policy id common_roles.yaml and other_roles.yaml
> ----
> cerbosctl get derived_roles --policy-id=common_roles.yaml,other_roles.yaml
> cerbosctl get dr --policy-id=common_roles.yaml,other_roles.yaml

Reviewer: ```suggestion
.Get derived roles policies having the ID `common_roles.yaml` and `other_roles.yaml`
----
cerbosctl get derived_roles common_roles.yaml other_roles.yaml
cerbosctl get dr common_roles.yaml other_roles.yaml
```

The other command examples need to be similarly updated as well.
